using-ide-vscode.md

```diff
@@ -208,7 +214,7 @@ types of code changes cannot be hot reloaded though:
 
 For these changes you can fully restart your application, without having to end
 your debugging session by running the **Flutter: Hot Restart** command from the
```

Comment:
session by -> session, by
Or rewrite, since this sentence is pretty long and could be clearer.